### DISCUSSION TITLE
Platform-specific dependency

### DISCUSSION THREAD

**Discussion by index1207** [microsoft/vcpkg#40311] [Q&A] [answered]
I'm trying to make my library cross-platform.
The problem is that the library's dependency only works on Windows.

Is there a way to implement platform-specific dependencies?

**Comment by Osyotr** [comment, marked as answer]
Yes, using `platform` expression. Example:
https://github.com/microsoft/vcpkg/blob/a2367ceec5f092d8777606ca110426cadd7ed7db/ports/qt5-webengine/vcpkg.json#L11

**Reply by index1207** [reply]
Thank you for your answer!
